### DISCUSSION TITLE
An error I'm not sure I understand

### DISCUSSION THREAD

**Discussion by huongngo-8** [featureform/featureform#928] [Q&A]
Hello! Not sure if this is the correct place to send this question, but I'm trying to use featureform and Pinecone to contextualize GPT-4 prompts and I'm running into this error: `OperationalError: table feature_variant has 14 columns but 16 values were supplied` after registering an entity and a feature. I'm following the Featureform MLOps Podcast Chat Bot notebook to do this. Thank you!

**Comment by sdreyer** [comment]
Hey @huongngo-8 , can you let me know what version of featureform you're using and if you were using a different version previously what that version was?

**Reply by huongngo-8** [reply]
Sure! I think I'm using version 1.10.0, which I think is the latest version, but please correct me if I'm wrong.

**Reply by huongngo-8** [reply]
@sdreyer Hi! Just wanted to follow up on this, thanks.

**Reply by sdreyer** [reply]
Did you happen to have a different version installed previously? Or is this your first time installing?
